lifecycle_test.go

```diff
@@ -148,6 +148,9 @@ func TestLifecycleFullFlow(t *testing.T) {
 	exportRb(t, tests.LcRbName2, number2, tempDir)
 	defer deleteExportedReleaseBundle(t, tests.LcRbName2)
 
+	// Test import
+	importRb(t, tests.LcRbName2, number2)
```

Comment:
If I'm not mistaken, the release bundle already exists before importing. Consider moving the import (and maybe export as well) to a separate test, to also avoid this test being too long